newIDE/electron-app/app/main.js

```diff
@@ -71,10 +71,12 @@ app.on('ready', function() {
     x: args.x,
     y: args.y,
     webPreferences: {
-      webSecurity: false, // Allow to access to local files
+      webSecurity: false, // Allow to access to local files,
+      nodeIntegration: true,
+      webviewTag: true
```

Comment:
I don't think we are (or want in the future) to use the `<webview>` tag, so you can omit this one :)

Reply:
Oh okay, done.